system/src/system_power_manager.cpp

```diff
@@ -146,10 +150,28 @@ void PowerManager::handleUpdate() {
   power_source_t src = g_powerSource;
   if (pwr_good) {
     uint8_t vbus_stat = status >> 6;
+    // LOG_DEBUG(INFO, "vbus_stat: 0x%x, usb state: %d, current limit: %d", vbus_stat, HAL_USB_Get_State(), power.getInputCurrentLimit());
     switch (vbus_stat) {
-      case 0x01:
+      case 0x01: {
+#if HAL_INCREASE_CHARGING_CURRENT_WHEN_POWERED_BY_VIN
+        // Workaround: 
+        // when Gen3 device is powered via VUSB, USB peripheral gets no power supply.
```

Comment:
Let's change the wording to 'powered by VIN'. I know the silkscreen on the PCB says VUSB, but it's confusing as VUSB usually comes from the USB port.